src/scripts.js

```diff
@@ -5,9 +5,13 @@ const userInfoSection = document.querySelector('.user-info-js')
 const dateSection = document.querySelector('.date-js')
 const waterSection = document.querySelector('.user-water-info-js')
 const waterSectionContainer = document.querySelector('.user-water-js')
-let waterDateSelection = document.querySelector('.selection-box-js')
+let waterDateSelection = document.querySelector('.selection-water-js')
```

Comment:
the let on lines 8/9 should be changed to 'const' — these are DOM elements that do not need to be altered or reassigned.